modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementFailureTest.java

```diff
@@ -0,0 +1,342 @@
+        String invalidRequestBody = readResource("invalid-discovery-config-request-body.json");
+        Response response = getResponseOfPostWithOAuth2(endpointURL, invalidRequestBody, m2mToken);
+        validateErrorResponse(response, HttpStatus.SC_BAD_REQUEST, "UE-10000");
+    }
+
+    @Test(dependsOnMethods = "testAddInvalidDiscoveryConfig")
+    public void testAddDiscoveryConfigUnauthorized() throws IOException {
+
+        String endpointURL = ORGANIZATION_CONFIGS_API_BASE_PATH + ORGANIZATION_DISCOVERY_API_PATH;
+        String requestBody = readResource("add-discovery-config-request-body.json");
+        Response response = getResponseOfPostWithOAuth2(endpointURL, requestBody, invalidM2MToken);
+        validateHttpStatusCode(response, HttpStatus.SC_UNAUTHORIZED);
+    }
+
+    @Test(dependsOnMethods = "testAddDiscoveryConfigUnauthorized")
+    public void testAddExistingDiscoveryConfig() throws IOException {
+
+        String endpointURL = ORGANIZATION_CONFIGS_API_BASE_PATH + ORGANIZATION_DISCOVERY_API_PATH;
+        String requestBody = readResource("add-discovery-config-request-body.json");
+        Response firstResponse = getResponseOfPostWithOAuth2(endpointURL, requestBody, m2mToken);
+        validateHttpStatusCode(firstResponse, HttpStatus.SC_CREATED);
+        Response secondResponse = getResponseOfPostWithOAuth2(endpointURL, requestBody, m2mToken);
+        validateErrorResponse(secondResponse, HttpStatus.SC_CONFLICT, "OCM-60003");
+    }
+
+    @Test(dependsOnMethods = "testAddExistingDiscoveryConfig")
+    public void testGetDiscoveryConfigUnauthorized() {
+
+        String endpointURL = ORGANIZATION_CONFIGS_API_BASE_PATH + ORGANIZATION_DISCOVERY_API_PATH;
+        Response response = getResponseOfGetWithOAuth2(endpointURL, invalidM2MToken);
+        validateHttpStatusCode(response, HttpStatus.SC_UNAUTHORIZED);
+    }
+
+    @Test(dependsOnMethods = "testGetDiscoveryConfigUnauthorized")
+    public void testAddInvalidDiscoveryAttributesToOrganization() throws IOException {
+
+        String endpointURL = ORGANIZATION_MANAGEMENT_API_BASE_PATH + ORGANIZATION_DISCOVERY_API_PATH;
+        String invalidRequestBody = readResource("add-invalid-discovery-attributes-request-body.json");
+        invalidRequestBody = invalidRequestBody.replace("${organizationID}", organizationIDs.get(0));
+        Response response = getResponseOfPostWithOAuth2(endpointURL, invalidRequestBody, m2mToken);
+        validateErrorResponse(response, HttpStatus.SC_BAD_REQUEST, "UE-10000");
+    }
+
+    @Test(dependsOnMethods = "testAddInvalidDiscoveryAttributesToOrganization")
+    public void testAddDiscoveryAttributesToNonExistingOrganization() throws IOException {
+
+        String endpointURL = ORGANIZATION_MANAGEMENT_API_BASE_PATH + ORGANIZATION_DISCOVERY_API_PATH;
+        String requestBody = readResource("add-discovery-attributes-request-body.json");
```

Comment:
This file has been used in multiple places, let's use a constant for this. And do the same for other duplicated string as well